docs/dev-run.js

```diff
@@ -64,7 +64,7 @@ portfinder.getPorts(2, {}, (portFinderErr, [docsPort, webpackPort]) => {
     process.exit(1);
   }
 
-  runCmd('webpack-dev-server', `nodemon --watch webpack --watch webpack.docs.js --exec webpack-dev-server -- --config webpack.docs.js --color --port ${webpackPort} --debug --hot --host ${ip.address()}`);
+  runCmd('webpack-dev-server', `nodemon --watch webpack --watch webpack.docs.js --exec webpack-dev-server -- --config webpack.docs.js --color --port ${webpackPort} --debug --hot --inline --host ${ip.address()}`);
```

Comment:
It was for cross browser testing from another machine: https://github.com/react-bootstrap/react-bootstrap/commit/9536fa6a13e4414b823720b498177405026c0d75